### PR TITLE
Support date shortcuts; see issue #13

### PR DESCRIPTION
**Pull request by kaat0**
Tests are currently missing and help is welcome.

### CODE REVIEW

File: src/Overpass.jl
Comment (kaat0) [inline src/Overpass.jl:1]: Doesn't the `current_date` needs to be in UTC for overpass API? That is the "Z" at the end of the time string.

**Comment by gwehrle** [comment]
I fixed the tests :tada: 
It was pretty hard to find out that adding TimeZones.jl modifies parsing of Date.jl library :face_exhaling: 

I'll add some tests for the new function soon.

**Comment by gwehrle** [comment]
I just saw that in Overpass turbo time calculation is done with constants of seconds. This means for durations like months or years this could lead to different behaviour. I still would stick to the current behaviour. 

https://github.com/tyrasd/overpass-turbo/blob/0ff477e87d947645a6f70a4d408edf42d1166306/js/shortcuts.ts#L44-L74

**Comment by gwehrle** [comment]
All tests are done. For me this feature is ready to merge.
What do you think @kaat0 ? Is there something missing?

**Comment by gwehrle** [comment]
Okay there are still two todos:
- [ ] remove Timezones dependency, instead use Dates now(UTC)
- [ ] remove Mocking dependency, as test code doesn't belong in production code

**Comment by gwehrle** [comment]
@kaat0 Thank you a lot!

**Pull request merged**